cli/azd/resources/ai-python/requirements.txt

```diff
@@ -0,0 +1,6 @@
+promptflow
+promptflow-azure
+promptflow-tools
+azure-ai-ml
```

Comment:
Have you consider pulling the ai-python files from some repo?  (instead of embedding it to azd binary?)

By not having versions for this libraries, are we getting the latest?  What if there are breaking changes when pulling latest? that would requiere releasing a new azd version :(  

Moving this program out of azd would be easier to react to it and have it updated w/o affecting azd

Or, have you consider creating python wheels?

Reply:
This is only intended to be a temporary solution until these features are natively available in the Azure SDK for Go and therefore would opt to keep them as embedded scripts within azd instead and treat them as any other piece of source code in the CLI.  They are not intended to be called directly by end users code and are only executed from within `azd`.